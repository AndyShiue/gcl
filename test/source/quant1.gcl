b := <| (+) i : 0 <= i < k : F i |>
n

{ true }
