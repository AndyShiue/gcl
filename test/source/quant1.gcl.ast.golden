b := ⟨ Σ i : (0 ≤ i) < k : F i ⟩
n := 0

{ true }